src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -13,6 +13,22 @@
      * числа 96, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        String reworkedString = text.replaceAll("[^\\d.-]", " ");
```

Comment:
```suggestion
        String reworkedString = text.replaceAll("[^\\d.-]+", " ");
```
and remove checking to ""

Reply:
Привіт!
дякую, не знав, що такий метод існує)
але в мене тожі виникає проблема і я не розумію звідки вона.
При трімі стрінгу на масив стрінгів в мене створюється один зайвий перший елемент (""), тому некоректна довжина масиву виходить
і далі всеодно потрібно перевіряти на ""